types/product/updates/ProductChangeSlugAction.raml

```diff
@@ -1,13 +1,17 @@
 #%RAML 1.0 DataType
 (package): Product
 (docs-uri): https://docs.commercetools.com/http-api-projects-products.html#change-slug
-
 type: ProductUpdateAction
 displayName: ProductChangeSlugAction
 discriminatorValue: changeSlug
 (postman-example): !include ../../../examples/Product/ProductChangeSlugAction.json
 properties:
   slug:
     type: LocalizedString
+    description: |-
+      Every slug must be unique across a project, but a product can have the same slug for different languages.
+      Allowed are alphabetic, numeric, underscore (`_`) and hyphen (`-`) characters.
+      Maximum size is {{ site.data.api-limits.slugLength }}.
```

Comment:
Here we should put in the actual value. This is a good example why parsing the HTML version might have been a better idea 😉 😁